### PR TITLE
added usage and paper

### PR DESCRIPTION
**Pull request by raul-ortega**
Whe have added an usage and the link to a paper describing ontoavida that has recently been published

### CODE REVIEW

File: ontology/ontoavida.md [hunk 1]
```diff
@@ -2,9 +2,13 @@
...
 id: ontoavida
 title: 'OntoAvida: ontology for Avida digital evolution platform'
+browsers:
+- title: Ontoavida HTML Browser
+  label: Human-readable (HTML)
+  url: https://owl.fortunalab.org/ontoavida/
 contact:
   email: fortuna@ebd.csic.es
-  github: miguelfortuna
+  github: fortunalab
```
Comment (cthoyt) [inline ontology/ontoavida.md:11]: it's OBO Policy to have a single person point of contact, so I would suggest reverting this change

Comment (raul-ortega) [reply ontology/ontoavida.md:11]: Dear Carlos, thank you very much for the note, it is our wish to modify the contact information from @miguelfortuna to @fortunalab becouse ontoavida is a work related to the lab and not to a single individual. Please, consider aproving this PR as it was done for PR [#2044](https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2044) after @miguelfortuna add a comment telling this information is right. Thanks so much.

Comment (cthoyt) [reply ontology/ontoavida.md:11]: Most OBO projects are indeed not built by a single individual, but nonetheless, a single person's contact information is required. Please take a look at [OBO Principle 11 "locus of authority"](https://obofoundry.org/principles/fp-011-locus-of-authority.html) that defines the explicit requirement:

> The contact email MUST NOT be a mailing list. The GitHub account MUST be for the individual designated as the locus of authority. If this person does not already have a GitHub account, we request that they [create one](https://github.com/join). Then, add the following to your [metadata file](https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology) (replacing with the correct email, name, and GitHub username):
>
> contact: email: foo@bar.com label: John Smith github: jsmith123

Comment (raul-ortega) [reply ontology/ontoavida.md]: ok, I understand and I apologize for not complying with the policy. I have reissued the contact.

Comment (miguelfortuna) [reply ontology/ontoavida.md]: That's fine. We have not changed the contact information. Please, accept the pull request. Thanks.

Comment (cthoyt) [reply ontology/ontoavida.md]: Thanks! @deepakunni3 has merge rights so after he gives it a check, it should be good to go.

Comment (deepakunni3) [reply ontology/ontoavida.md]: Thanks @cthoyt for the review! 👍

**Review by deepakunni3 — APPROVED** [review]

**Pull request merged**